src/client/signatures/canonical_json/README.md

```diff
@@ -0,0 +1,45 @@
+# Canonical JSON library (Remote-Settings)
+
+  Canonical JSON is a variant of JSON in which each value has a single,
+  unambiguous serialized form. This provides meaningful and repeatable hashes
+  of encoded data.
+
+Compared to serde_json::to_string
+- String contents are not guaranteed be parsable as UTF-8. Be aware that encoded data may contain escaped unicode characters.
+- Minus Zero is disallowed
+- Object keys must appear in lexiographical order and must not be repeated.
+
+[Link to the Canonical Spec](spec.txt)
+
+Comparsion to other Canonical JSON implementation - https://github.com/zmanian/canonical_json
+
+- This Canonical JSON library supports floating-point numbers, exponents, and can convert Unicode characters into Unicode escape sequences
```

Comment:
Which is "This"?  Our implementation I hope?

Reply:
yes 😄